core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -3251,6 +3251,44 @@ paths:
           required: true
           schema:
             $ref: '#/definitions/DataRepoSnapshot'
+    get:
+      requestBody:
+        description: |
+          Parameters for enumerating snapshots.
+        content:
+          application/json:
+            schema:
+              $ref: '#/definitions/EnumerateSnapshotRequestBody'
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/DataRepoSnapshotList'
+        '404':
+          description: Workspace or snapshot not found
```

Comment:
just "workspace not found"? I assume this will return an empty array if no snapshots are found.

Reply:
updated - it could also be "user lacks permissions"